docs/timeseries.md

```diff
@@ -1,122 +1,275 @@
+```
+
+#### TimeSeries.merge(info, [series1, series2, ...]) [Static]
+
+Returns a new TimeSeries that merges a list of TimeSeries together. The common uses for this are:
+
+ * Join a list of `TimeSeries` together that have different columns.
+ * Append together a list of `TimeSeries` with the same columns, but different times.
+
+The `info` passed into the `merge()` function is an Object which is used to create the new `TimeSeries`. It contains a name, index (optional), and meta data.
+
+There are some rules surrounding the use of `merge()`.
+
+If we consider each row of each `TimeSeries` with the same `Date` (or `Index`, or `TimeRange`), then we have a list of `Events` (or `IndexedEvents`, or `TimeRangeEvents`). This list needs to be reduced. To do this the events themselves are merged using `Event.merge()`. This operation will not attempt to reduce values which have the same column, for the same time, so for each of these lists of events there should be no shared columns. In other words, you can merge a `TimeSeries` with columns "a" and "b" with a `TimeSeries` with a column "c", but not with a `TimeSeries` with a column "a" (if the two `TimeSeries` overlap their times).
+
+For example, first we create two `TimeSeries`, one with a "in" column and one with an "out" column:
+
+```javascript
+const inTraffic = new TimeSeries({
+    name: "in",
+    columns: ["time", "in"],
+    points: [
+        [1400425947000, 52],
+        [1400425948000, 18],
+        [1400425949000, 26],
+        [1400425950000, 93],
+    ]
+});
+
+const outTraffic = new TimeSeries({
+    name: "out",
+    columns: ["time", "out"],
+    points: [
+        [1400425947000, 34],
+        [1400425948000, 13],
+        [1400425949000, 67],
+        [1400425950000, 91],
+    ]
+});
+```
+
+We can them merge them:
+
+```javascript
+var trafficSeries = TimeSeries.merge({name: "traffic"}, [inTraffic, outTraffic]);
+```
+
+The result will look like this:
+
+```json
+{
+    "name":"traffic",
+    "columns":["time", "in", "out"],
+    "points":[
+        ["2014-05-18T15:12:27.000Z", 52, 34],
+        ["2014-05-18T15:12:28.000Z", 18, 13],
+        ["2014-05-18T15:12:29.000Z", 26, 67],
+        ["2014-05-18T15:12:30.000Z", 93, 91]
+    ]
+}
+```
+
+### Comparison static functions
+
+One of the nice things about the `TimeSeries` representation in Pond is that it is built on top of immutable data structures. For instance, if you have a `TimeSeries` and modify it, those `TimeSeries` will now have a different reference and noting that they have changed is trivial.
+
+#### TimeSeries.equals(series1, series2) [Static]
 
-### Comparison
+Will check that the internal structures of the `TimeSeries` are the same reference. If you use the copy constructor, they will be the same.
 
-One of the nice things about the TimeSeries representation in Pond is that it is built on top of immutable data structures. As a result, determining if a series is different from before is trivial.
```

Comment:
Might be worth keeping this or some form of it.  Knowing that comparisons are efficient due to the underlying immutable data structures is a useful.